plugins/cryptolib/cryptolib.cpp

```diff
@@ -525,16 +541,16 @@ CRYPTOLIB_API void CRYPTOLIB_CALL clPKISign(size32_t & __lenResult,void * & __re
     }
 }
 
-CRYPTOLIB_API void CRYPTOLIB_CALL clPKISignLFN(ICodeContext * ctx,
+CRYPTOLIB_API void CRYPTOLIB_CALL clPKISignLFN2(ICodeContext * ctx,
                                         size32_t & __lenResult,void * & __result,
                                         const char * pkalgorithm,
                                         const char * privatekeyLFN,
-                                        const char * passphrase,
+                                        size32_t lenPassPhrase,const void * passphrase,
                                         size32_t lenInputdata,const void * inputdata)
 {
     verifyPKIAlgorithm(pkalgorithm);
     IUserDescriptor * udesc = ctx->queryUserDescriptor();
-    Owned<IDigitalSignatureManager> pDSM = g_DSMCache.getInstance(pkalgorithm, nullptr, nullptr, privatekeyLFN, nullptr, passphrase, true, udesc);
+    Owned<IDigitalSignatureManager> pDSM = g_DSMCache.getInstance(pkalgorithm, nullptr, nullptr, privatekeyLFN, nullptr, (const char *)passphrase, true, udesc);
```

Comment:
Also needs to pass lenPassPhrase (and more examples below)

Reply:
Done